Revit_Core_Engine/Query/OpeningSurface.cs

```diff
@@ -225,13 +233,20 @@ private static List<ISurface> OpeningSurfaces_Curtain(this FamilyInstance family
         }
 
         /***************************************************/
+        private static List<ISurface> GetUnhostedOpeningGeometry(this FamilyInstance familyInstance, Document doc, RevitSettings settings = null)
+        {
+            List<ISurface> surfaces = new List<ISurface>();
```

Comment:
Isn't it worth a warning message about empty surface?

Reply:
Warning added in latest commit.